rfc3676.c

```diff
@@ -322,7 +322,7 @@ int rfc3676_handler(struct Body *a, struct State *s)
 
     /* for DelSp=yes, we need to strip one SP prior to CRLF on flowed lines */
     if (delsp && !fixed)
-      buf[--buf_len] = '\0';
+      buf[buf_len - 1] = '\0';
```

Comment:
Hmm... that's an interesting change.
Did `cppcheck` suggest this, or is it your work?

Reply:
No, cppechek suggested that the scope of `buf_len` could be reduced.
I noticed that only there the value got was mutated. I replace it so that `buf_len` can be declared as `const`.
Since the value is not used afterwards, it should not cause problems.

Reply:
nice :-)